sys/linux/auto.txt

```diff
@@ -6495,12 +8598,68 @@ udp_timeout_nla_policy$auto_nf_conntrack_proto_udp [
 	CTA_TIMEOUT_UDP_REPLIED		nlattr[CTA_TIMEOUT_UDP_REPLIED, int32]
 ] [varlen]
 
+unewset$auto_record {
+	sig	array[int32, 1]
```

Comment:
sigset_t should contain int64 on a 64-bit platform.
Also `array[T, 1]` is always better represented as just `T`.

Reply:
Done!